clients/unshaded/src/main/java/tachyon/client/ReadType.java

```diff
@@ -15,29 +15,32 @@
 
 package tachyon.client;
 
+import tachyon.annotation.PublicApi;
+
 /**
- * Different read types for a TachyonFile.
+ * Convenience modes for commonly used read types for a TachyonFile.
  *
- * As of 0.8, replaced by {@link TachyonStorageType} and {@link UnderStorageType}
+ * For finer grained control over data storage, advanced users may specify
+ * {@link tachyon.client.TachyonStorageType} and {@link tachyon.client.UnderStorageType}.
  */
-@Deprecated
+@PublicApi
 public enum ReadType {
   /**
-   * Read the file and but do not cache it explicitly.
+   * Read the file and skip Tachyon storage. This mode will not alter any data distribution
```

Comment:
I'm a little confused by the second sentence, is it talking about how `NO_CACHE` also means `NO_PROMOTE`?